### ISSUE TITLE
accept character input based on special param

### ISSUE THREAD

**Issue by maelle** [lvaudor/glitter#62]
e.g. `accept_string`. Might make things easier to program with.

**Comment by lvaudor** [comment]
I'm not sure what you're referring to? do you have an example?

**Comment by maelle** [comment]
So you could have

`spq_summarise("nbdoc = n(unique(document))")`

this way someone could form the call to `spq_summarise` in a function/loop more easily.

**Comment by maelle** [comment]
so the rule would be

- character of class spql -> do nothing.
- other character, not character -> parse as R-SPARQL DSL.

**Comment by lvaudor** [comment]
Indeed. But I think this would be a rather "glitter use -advanced level" kind of thing. From my own use of glitter I'd say the need for programming will arise first in the construction of the basic query itself i.e. the calls to spq_add(). I tried to address it a little with the possibility to use R objects in the triplets with the {thingy} notation but maybe there are ways to improve this (allowing users or showing users how to purrr::map() some things by themselves for instance). Probably this will be clearer the more users (and use examples) we get...

**Comment by maelle** [comment]
Actually to use purrr::map() you need the string input. It's quite easy to add the string input, I'll add it but without parameter, just so you see how it'd be.
